tests/test_notes_reminders_storage.py

```
@@ -0,0 +1,151 @@
+    "get_custom_reminder_by_id",
+    "set_custom_reminder_active",
+    "delete_custom_reminder",
+    "get_all_active_custom_reminders",
+]
+
+
+def test_both_backends_expose_the_same_methods():
+    from database import DatabaseManagerMongo
+    import database
+
+    sql_manager = getattr(database, "DatabaseManager")
```

Ruff (0.16.1)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)